docs/docsgen/source/technical/float8.rst

```diff
@@ -0,0 +1,99 @@
+
+.. _onnx-detail-float8:
+
+======================
+Float stored on 8 bits
+======================
+
+.. contents::
+    :local:
+
+Papers
+======
+
+Two papers have been published in 2022 to introduce floats
+stored on a byte as opposed to float 32 stored on 4 bytes.
+The float precision is much lower but the training precision
+does not suffer too much.
+
+`FP8 Formats for Deep Learning <https://arxiv.org/abs/2209.05433>`_
+from NVIDIA introduces two types following
+`IEEE specifciations <https://en.wikipedia.org/wiki/IEEE_754>`_.
+First one is E4M3, 1 bit for the sign, 4 bits for the exponents and 3
+bits for the mantissa. Second one is E5M2, 1 bit for the sign,
+3 bits for the exponents and 2 for the mantissa. The first types
+is mostly used for the coefficients, the second one for the gradient.
```

Comment:
Is "coefficient" the right word? Is it supposed to be for weights?

Reply:
yes, in French, they are synonyms. I'll change that.

Reply:
done